colcon_core/command.py

```diff
@@ -86,7 +86,6 @@ def register_command_exit_handler(handler):
 
     :param handler: The callable
     """
-    global _command_exit_handlers
     if handler not in _command_exit_handlers:
         _command_exit_handlers.append(handler)
```

Comment:
I'm curious if this runs successfully in terms of finding the correct reference to the global variable. From my understanding it should fail since `_command_exit_handlers` is not defined in the function. (But there might be some python juju magic in place to make it  kinda work)